swift/probe/build.bash

```diff
@@ -0,0 +1,11 @@
+# run with bash -e on an m1 mac
+swiftc -Osize ./Sources/Moonlight/moonlight.swift -o moonlight_darwin-x86_64 -target x86_64-apple-macos10.13 && strip moonlight_darwin-x86_64
+
+swiftc -Osize ./Sources/Moonlight/moonlight.swift -o moonlight_darwin-arm64 -target arm64-apple-macos10.13 && strip moonlight_darwin-arm64
```

Comment:
I don't like this file, as it's a specific install script from a specific POV (arm/Mac). see readme comment on making the process described clearer from each perspective.

Reply:
Removed